include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_utils.h

```diff
@@ -850,15 +850,16 @@ __bypass_sycl_kernel_not_supported(const sycl::exception& __e)
 {
     // The SYCL spec compliant solution would be to compare __e.code() and sycl::errc::kernel_not_supported
     // and rethrow the encountered exception if the two do not compare equal. However, the icpx compiler currently
-    // returns a generic error code in violation of the SYCL spec which has a value of 7. If we are using the Intel
-    // compiler, then compare the value of the error code. Otherwise, assume the implementation is spec compliant.
+    // returns a sycl::errc::build in violation of the SYCL spec. If we are using the Intel compiler, then compare
+    // to this error code. Otherwise, assume the implementation is spec compliant.
+    const std::error_code __kernel_not_supported_ec =
```

Comment:
Let's include `<system_error>` to make sure `std::error_code` is available.

Reply:
Done